chalice/chalicelib/config.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+import os
+import boto3
+import hca
+
+from dcplib.aws_secret import AwsSecret
+from tweak import Config
+from cloud_blobstore.s3 import S3BlobStore
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIGURATION_FILE = os.path.join(BASE_DIR, "config.json")
+
+# Load configuration
+with open(CONFIGURATION_FILE) as f:
+    configuration = json.load(f)
+    hca_client_host = configuration["hca_host"]
+    ms_secret_name = configuration["ms_secret_name"]
+
+# Default directory for all temp files
+TEMP_DIR = "/tmp"
```

Comment:
Yes, I do use `tempfile`. The thing is in lambda, I am only allowed to write to a directory called `tmp/`. Thus, even if I use `tempfile` to create files or directories, I have to specify the root directory for them is `tmp/`, which is where this variable `TEMP_DIR` being used for.

Reply:
Okay cool, sounds fine to me. I only ask because I'm not quite sure what the exact promise made by lambda is. Is `/tmp` writable or is it, say, the value of `$TMPDIR`.

Reply:
It is `/tmp`, not the value of `$TMPDIR`. I have done some experiments on it, and they work correctly.